commit 53e8558837be58c1d44d50ad87247a8c56c95c13
Author: Chen Yu <yu.c.chen@intel.com>
Date:   Wed Dec 22 12:32:34 2021 +0800

    ACPI: tools: Introduce utility for firmware updates/telemetry
    
    Introduce a user space tool to make use of the interface exposed by
    Platform Firmware Runtime Update and Telemetry drivers.
    
    It can be used for firmware code injection, driver updates and
    to retrieve platform firmware telemetry data.
    
    Tested-by: Hongyu Ning <hongyu.ning@intel.com>
    Signed-off-by: Chen Yu <yu.c.chen@intel.com>
    [ rjw: Subject and changelog edits ]
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

diff --git a/tools/power/acpi/Makefile.rules b/tools/power/acpi/Makefile.rules
index 1d7616f5d0ae..b71aada77688 100644
--- a/tools/power/acpi/Makefile.rules
+++ b/tools/power/acpi/Makefile.rules
@@ -9,7 +9,7 @@ objdir := $(OUTPUT)tools/$(TOOL)/
 toolobjs := $(addprefix $(objdir),$(TOOL_OBJS))
 $(OUTPUT)$(TOOL): $(toolobjs) FORCE
 	$(ECHO) "  LD      " $(subst $(OUTPUT),,$@)
-	$(QUIET) $(LD) $(CFLAGS) $(LDFLAGS) $(toolobjs) -L$(OUTPUT) -o $@
+	$(QUIET) $(LD) $(CFLAGS) $(toolobjs) $(LDFLAGS) -L$(OUTPUT) -o $@
 	$(ECHO) "  STRIP   " $(subst $(OUTPUT),,$@)
 	$(QUIET) $(STRIPCMD) $@